doc/configuration.rst

```diff
@@ -279,6 +279,37 @@ An example publish prefix is as follows:
 
 By default, no prefix is used. See also |confluence_publish_postfix|_.
 
+confluence_add_secnumbers
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Add section numbers to section titles in pages if ``doctree`` uses the
+``:numbered:`` option. This causes Confluence to display the pages in the correct order in the page tree, since it orders them using lexicographical sorting. By default, this is disabled:
```

Comment:
I am a bit hesitant to promote the idea that using the `confluence_add_secnumbers` option "causes Confluence to display the pages in the correct order". Ordering is a side effect of this feature and there is no guarantee that a publish event will enforce ordering (especially if a node in a page tree is already configured for manual ordering).

If we really want to promote the idea that this option can potentially assist in page ordering, I believe it would be best outlined in a new section under the "tips" documentation.

_Side note:_ it would be ideal if the 80 character column was used for these documents when possible

Reply:
I'm also not sure if this ordering will work the same in all situations, so I removed the text.